Cargo.toml

```diff
@@ -6,4 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-csv = "1.1"
\ No newline at end of file
+csv = "1.1"
+anyhow = "1.0"
```

Comment:
[Anyhow is a very useful crate.](https://crates.io/crates/anyhow)

When we need to return errors that don't need to be `match`ed on we can use anyhow. Its a very nice way to return errors. `Box<dyn Error>` is also OK if we want to avoid an import, but its not very often you will see this.